src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -176,22 +177,32 @@ public WebSocketsTransport(WebSocketOptions options, Channel<byte[]> application
                     {
                         try
                         {
-                            _logger.SendPayload(_connectionId, buffer.Length);
+                            _logger.SendPayload(_connection.ConnectionId, buffer.Length);
+
+                            var webSocketMessageType = _options.WebSocketMessageType;
```

Comment:
I left it so that the user can set the WebSocketMessageType when using HttpConnection. I don't like this setting though at all since it is very general and applies to all WebSockets.  I am happy to remove if we are OK with always using the feature.